code/modules/mob/living/carbon/human/species_types/synths.dm

```diff
@@ -132,3 +134,30 @@
 					speech_args[SPEECH_SPANS] |= SPAN_CLOWN
 				if (/datum/species/golem/clockwork)
 					speech_args[SPEECH_SPANS] |= SPAN_ROBOT
+
+/datum/species/synth/spec_revival(mob/living/carbon/human/H)
+	H.grab_ghost()
+	H.visible_message("<span class='notice'>[H]'s eyes snap open!</span>", "<span class ='boldwarning'>You can feel your limbs responding again!</span>")
+
+/datum/species/synth/proc/handle_snowflake_code(mob/living/carbon/human/H, datum/species/S) //I LITERALLY FUCKING HATE ALL OF YOU. I HATE THE FACT THIS NEEDS TO EXIST.
+	switch(S.id)
+		if("felinid")
+			if(H.dna.features["tail_human"] == "None")
+				H.dna.features["tail_human"] = "Cat"
+			if(H.dna.features["ears"] == "None")
+				H.dna.features["ears"] = "Cat"
+			if(H.dna.features["ears"] == "Cat")
+				var/obj/item/organ/ears/cat/ears = new
+				ears.Insert(H, drop_if_replaced = FALSE)
+			else
+				mutantears = /obj/item/organ/ears
+			if(H.dna.features["tail_human"] == "Cat")
+				var/obj/item/organ/tail/cat/tail = new
+				tail.Insert(H, drop_if_replaced = FALSE)
+			else
+				mutanttail = null
+		if("lizard")
+			if(DIGITIGRADE in species_traits)
+				var/mob/living/carbon/C = H
+				default_features["legs"] = "Digitigrade Legs"
+				C.Digitigrade_Leg_Swap(FALSE)
```

Comment:
cant you just call update_bodyparts or whatever the proc is called to regenerate body parts?

Reply:
haha no because of insanely cursed bodypart code